app/javascript/availability/components/snippet.jsx

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+
+const Snippet = ({data}) => {
+    const librariesText = () => {
```

Comment:
```
   const isSnippet = () => {
        ...
        return ...
    };

    if (!isSnippet()) {
        return null;
    }
```

@nicktiberi Do you think it makes sense to have this one with the same approach you had for some of the others?

Reply:
@banukutlu - I changed this logic a bit so that it returns early if there are no copies available.